src/gsy_e/gsy_e_core/simulation.py

```diff
@@ -386,16 +393,22 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
                 self.handle_slowdown_and_realtime(tick_no)
                 self.tick_time_counter = time()
 
+                if self.is_stopped:
+                    log.info(f"Received stop command for "
+                             f"configuration id {gsy_e.constants.CONFIGURATION_ID} and "
```

Comment:
Can you please set this as `log.error` in order to see it in the kibana logs? This will make the debugging a lot easier. 
Also, can you pleas add the `gsy_e.constants.CONFIGURATION_ID` also to the start log in `src/gsy_e/gsy_e_core/rq_job_handler.py:14`? Thanks a lot!